tern-project.js

```diff
@@ -46,11 +46,22 @@ module.exports = function(root) {
             mod.initialize(ternRoot);
     });
 
-    // TODO load defs
+    // load defs
+    const defs = (project.libs || []).reduce((acc, lib) => {
+        const file = find('defs', lib, '.json');
+        if (file === undefined) {
+            // eslint-disable-next-line no-console
+            console.error(`Failed to load tern lib ${lib}`);
+            return;
+        }
+        acc.push(file);
+        return acc;
+    }, []);
```

Comment:
IMO this would be easier to read as a for loop
```suggestion
    const defs = [];
    for (const lib of (project.libs || [])) {
        const file = find('defs', lib, '.json');
        if (file === undefined) {
            // eslint-disable-next-line no-console
            console.error(`Failed to load tern lib ${lib}`);
            continue;
        }
        defs.push(file);
    }
```

Reply:
Lol, did not notice the `Commit suggestion` button!

Fixed with: df77191